docs/meadowlark-software-upgrade.md

```diff
@@ -0,0 +1,8 @@
+# Upgrade Meadownlark PC software version
+
+To upgrade your "PC software version" use these steps:
+
+- From "Add and remove programs", remove CellDrive5000 and "National Instruments Software".
+- From "Device manager", open the "Meadowlark Optics" group, right click `mlousb`, click "Uninstall device", check "Delete the driver software for this device", and click "Uninstall". Uninstall `Meadowlark Optics D5020 LC Driver` following the same steps.
+- Using the USB stick provided by Meadowlark, reinstall the USB drivers and CellDrive5000. 
+- **Upgrading users:** you will need to reinstall the Meadowlark device to your micromanager configuration file, because the device driver's name has changed to `MeadowlarkLC`. 
```

Comment:
Similar to my comment on #394 - these are instructions for users upgrading from recOrder 0.3 to recOrder 0.4, which is where the name change took place. This should move to the installation guide rather this Meadowlark software upgrade guide

Reply:
Thanks @ieivanov. I've updated the `0.4.0` docs by:
- moving the PC software version instructions to the installation guide
- moving the "Upgrading users" instruction to a more appropriate place, i.e. near the instructions where you're setting up your micromanager configuration
- clarifying the new device driver's name and updating the names to `MeadowlarkLC` everywhere.